local-rest-scorer/README.md

```diff
@@ -20,7 +20,22 @@ java -Dmojo.path={PATH_TO_MOJO_PIPELINE} -jar build/libs/local-rest-scorer-{YOUR
 
 To run the local scorer with shapley contribution enabled
 ```bash
+# Equivalent to using below and setting `-Dshapley.types.enabled=ALL`
 java -Dmojo.path={PATH_TO_MOJO_PIPELINE} -Dshapley.enable=true -jar build/libs/local-rest-scorer-{YOUR_CURRENT_VERSION}-boot.jar
+
+# SHAPLEY_TYPE can be one of [ALL, TRANSFORMED, ORIGINAL]
+java -Dmojo.path={PATH_TO_MOJO_PIPELINE} -Dshapley.types.enabled={SHAPLEY_TYPE} -jar build/libs/local-rest-scorer-{YOUR_CURRENT_VERSION}-boot.jar
```

Comment:
Ah, I see now what you mean re: not parsing env vars.  Great point.

https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getenv(java.lang.String)
In case you want to check for env vars.

It's [good practice](https://12factor.net/config).  Plus we can't reliably pass in arbitrary system properties from MLOps, but we can for envVars since it's standard method of configuration and not specific to Java.

Reply:
ya... I realized as much as I was reading the previous implementation. Imho would be nice to have as another refactor maybe... I think I will separate reading from env var to a separate pr, simply because its only partially related to this.